language/move-prover/bytecode/src/read_write_set_analysis.rs

```diff
@@ -83,12 +83,13 @@ impl ReadWriteSetState {
         accesses: AccessPathTrie<Access>,
         actuals: &[AbsAddr],
         type_actuals: &[Type],
+        temp_indices: &[TempIndex],
```

Comment:
Not sure if this is the bug, but something I noticed we shouldn't need `actuals: &[AbsAddr]` here anymore (and same with `apply_summary`)--the code can read the actual from `sub_map` using the temp indexes.

Reply:
Good point, thanks! I'll refactor that!